### ISSUE TITLE
Any estimate on the date of next release?

### ISSUE THREAD

**Issue by sahil87** [facebook/react-360#767]
Hi,
We heavily depend on react-360 for our projects. Having no visibility on when the next version will be released is hurting our ability to decide how to proceed.
Can you give us a rough estimate of when you are planning to make the next release? (Or is there a branch that we can start experimenting with now?)
Thanks in advance!

**Comment by tomyitav** [comment]
+1.
Waiting for hooks support and other improvements.

**Comment by jrjones** [comment]
Yeah, the vibe I'm getting is that Facebook has abandoned this before the open source community had a chance to take things over. I'm looking at Babylon, A-Frame, etc. Hoping I'm wrong about that., since I strongly prefer React-360.
